paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/UpdatePaymentMethodInteractor.kt

```diff
@@ -20,6 +21,7 @@ internal interface UpdatePaymentMethodInteractor {
     val canRemove: Boolean
     val displayableSavedPaymentMethod: DisplayableSavedPaymentMethod
     val screenTitle: ResolvableString?
+    val cardBrandFilter: CardBrandFilter
```

Comment:
ideally, all of the card brand filtering logic would live in DisplayableSavedPaymentMethod, and then we wouldn't need to add it here. It's hard to refactor the existing screen for that change though, so I'm going to do this once we remove the feature flag and remove the old screen